htsohm/simulation/helium_void_fraction.py

```diff
@@ -37,8 +37,10 @@ def parse_output(output_file):
                 results['vf_helium_void_fraction'] = float(line.split()[4])
             except IndexError:
                 print()
-    print("\nVOID FRACTION :   %s\n" % (results['vf_helium_void_fraction']))
-
+    try:
+        print("\nVOID FRACTION :   %s\n" % (results['vf_helium_void_fraction']))
+    except KeyError:
+        print("\nERROR PARSING VOID FRACTION DATA.")
```

Comment:
This is totally fine, but I'd be more inclined to use the pattern:
if 'vf_helium_void_fraction' in results:
    print('stuff')
else:
   print('error')